content/en/docs/distributions/ibm/deploy/OWNERS

```diff
@@ -1,7 +1,5 @@
 approvers:
-  - animeshsingh
```

Comment:
Hi @yhwang, maybe we can do the owner update in another PR. Animesh and shawn no long work at IBM so I can promote you as the new approver as well.

Reply:
sure, let me remove the changes for the OWNERS file in this PR for now